integration-tests/bot/jail_test.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python3
+import docker
+import json
+import requests
+import datetime
+import time
+from chainrpc import RPC, Blockchain
+class Program :
+    def __init__(self) :
+        self.rpc = RPC()
+        self.blockchain = Blockchain()
+
+        self.server="http://localhost:26657"
+        # wallet a
+        self.node0_address = ""
+        self.node0_mnemonics= ""
+
+        # wallet b
+        self.node1_address = ""
+        self.node1_mnemonics=""
+        self.headers = {
+            'Content-Type': 'application/json',
+        }
+
+    def get_containers(self) :
+        client = docker.from_env()
+        containers= client.containers.list()
+        ret= {}
+        for container in containers:
+            id = container
+            #ret[id.name]= id.id
+            ret[id.name]= container
+        return ret
+        
+
+    #show_containers()
+    # tendermint rpc
+
+
+
+    def get_staking_state(self,name, passphrase, addr):
+        return self.rpc.staking.state(addr, name)
+       
+
+    def create_staking_address(self,name, passphrase):
```

Comment:
Better to remove the unused passphrase argument.

Reply:
ok, i'll modify